.github/workflows/test.yml

```diff
@@ -34,15 +34,19 @@ jobs:
   test:
     strategy:
       matrix:
-        terraform_version: ["1.1.2", "1.4.0"]
+        terraform_version:
+          - "0.13.7"
+          - "0.15.5"
+          - "1.1.2"
+          - "1.5.3"
         # platform: [ubuntu-latest, macos-latest, windows-latest]
         platform: [ubuntu-latest]
-        # test old terraform versions only on ubuntu-latest
-        include:
-          - terraform_version: "0.15.5"
-            platform: ubuntu-latest
-          - terraform_version: "0.13.7"
-            platform: ubuntu-latest
+        # # test old terraform versions only on ubuntu-latest
```

Comment:
Should we remove that?

Reply:
Ended up removing everything commented out. We can add it back if we ever need to test windows etc